tomviz/Utilities.h

```diff
@@ -18,6 +18,7 @@
 
 // Collection of miscellaneous utility functions.
 
+#include "vtkPython.h" // must be first
```

Comment:
This shouldn't be needed now you moved Python utilities to their own file should it?